### ISSUE TITLE
add sound recording?

### ISSUE THREAD

**Issue by poVoq** [dmarcos/aframe-motion-capture-components#41]
Would be cool to have also sound recording for making a VR podcast or recording a VR meeting in the new Hubs VR space.

**Comment by dmarcos** [comment]
Good idea but it’s probably better to have a separate component that only handles audio recording
